site/docs/axis.md

```diff
@@ -38,64 +38,44 @@ Axis properties can be customized by setting `axis` to an axis property object.
-| ticks         | Number        | A desired number of ticks, for axes visualizing quantitative scales. The resulting number may be different so that values are "nice" (multiples of 2, 5, 10) and lie within the underlying scale's range. <span class="note-line">__Default value:__  derived from [axis config](config.html#axis-config)'s `ticks`. If the `ticks` config is undefined (default),  `ticks` is `5` for x-scale without binning, and `10` otherwise.</span>  |
-| tickColor		| String 		| The color of the axis's tick. |
-| tickLabelColor| String 		| The color of the tick label. |
-| tickLabelFont	| String 		| The font of tick label (e.g., `Helvetica Neue`). |
-| tickLabelFontSize	| Number 		| The font size of label, in pixels. <span class="note-line">__Default value:__ 10. </span> |
-| tickPadding   | Number        | The padding, in pixels, between ticks and text labels.|
-| tickSize      | Number        | The size, in pixels, of major, minor and end ticks. <span class="note-line">__Default value:__  derived from [axis config](config.html#axis-config)'s `tickSize` (`6` by default). </span>|
-| tickSizeMajor | Number        | The size, in pixels, of major ticks.|
-| tickSizeMinor | Number        | The size, in pixels, of minor ticks.|
-| tickSizeEnd   | Number        | The size, in pixels, of end ticks.|
-| tickWidth   	| Number        | The width, in pixels, of ticks.|
-| values        | Number[] &#124; [DateTime[]](transform.html#datetime) | Explicitly set the visible axis tick values. |
-
-### Title
-
-| Property      | Type          | Description    |
-| :------------ |:-------------:| :------------- |
-| title         | String        | A title for the axis. <span class="note-line">__Default value:__  derived from the field's name and transformation function applied e.g, "field_name", "SUM(field_name)", "BIN(field_name)", "YEAR(field_name)".</span> |
-| titleColor	| String 		| The color of the title. |
-| titleFont		| String 		| The font of title (e.g., `Helvetica Neue`). |
-| titleFontWeight	| Number 	 	|  The font weight of title (e.g., `bold`). |
-| titleFontSize	| Number 		| The font size of title, in pixels. <span class="note-line">__Default value:__ 10. </span> |
-| titleOffset   | Number        | The offset (in pixels) from the axis at which to place the title.|
-| titleMaxLength  | Integer     | Max length for axis title when the title is automatically generated from the field\'s description. <span class="note-line">__Default value:__  automatically determined based on the cell size (`config.cell.width`, `config.cell.height`) and `characterWidth` property.</span> |
-| characterWidth  | Integer       | Character width for automatically determining the value of `titleMaxLength`. <span class="note-line">__Default value:__  derived from [axis config](config.html#axis-config)'s `characterWidth` (`6` by default). </span> |
+### Axis Properties
```

Comment:
The intro paragraph above is outdated.   Now people can customize axis either via Axis Properties (which is applied to a specific config.) or via Axis Config (which get applies to all Axis).

Please update the intro paragraph accordingly.  You might want to add an introduce paragraph for "Axis Properties" and "Axis Config" as well.